### ISSUE TITLE
Character Died (Lost Health, Gold, Item, etc)

### ISSUE THREAD

**Issue by Kimazon** [HabitRPG/habitica#8068]
- UUID:  4738d199-99fb-497c-a462-4521c198564a
- Browser: Chrome
- OS: Mac OSX 10.9.5

I just died (lost all my health, gold, etc) and I have no idea why? I checked my party messages and no one caused enough damage to seriously injure me. Yet, I lost all of my health in one hit. Any idea what might have happened?

I asked in the Help Guild and was directed here by Alys who looked at the logs but couldn't determine why I had died at that time. 

I don't believe I had clicked any tasks when it happened, though I do believe I had reloaded the page. All of my party had just had their chron time. Whereas mine is set for a few hours after theirs. It happened right about the time of their chron, but no one had caused any major damage, and my own chron had not happened yet, so there was no reason I can see why I would have lost health. It just suddenly said I had lost all 50 of my health at once.

I believe this may have happened at least once (maybe twice) before? At the time I thought it was a bit strange that I had died, but I had not been paying complete attention to how much health I had before. I just brushed it off thinking I had missed something but I remember thinking at the time that I really didn't think that my health was low enough for the amount of damage my party had caused to have killed me so suddenly. I wish I had more details, but I still thought it might be worth mentioning.

**Comment by Alys** [comment]
@Kimazon I've looked into this in a lot more detail now and I think what happened does mostly make sense. Our logs show that your cron ran at 2016-09-24 09:43 UTC (2016-09-24 early morning in your timezone) and just before then your health was actually 21, rather than 50 (see below for more about that). You had a few Dailies incomplete so the boss did 15.7 damage to the party, including to you. The Dailies themselves would have done about half that much damage to you by themselves, producing enough total damage to reduce your health from 21 to below zero.

On your cron before that, your health had been about 43 just before the cron ran, and your missed Dailies caused the boss to do 14.9 damage to the party, and the Dailies themselves again did about half that damage to you directly.

So all of that tallies up correctly. The only troubling part about this why you weren't seeing that your health was lower than 50, especially since it had been beneath 50 for at least a day. Can you tell me if you were using the website or mobile app (iOS or Android?) or both, over the past couple of days? It seems that it might have been showing you that you had full health when you didn't. Or have you been buying health potions or having healing skills cast on you that appeared to be increasing your health (but in reality your purchases or the skills might not have been saved permanently)? Have you noticed anything odd about your health or Habitica in the past couple of days? For example, have you noticed any times when you took some action (e.g., buying something or marking a task complete) and then later found that the action had been reversed? Have you been seeing any error messages on the site or in the app?

**Comment by Kimazon** [comment]
Thanks for looking into it, Alys.

I do use a combination of the app and website, but mostly the website. 

I have bought some potions over the last week, but I couldn't tell you when. I mostly just buy them if I notice my health is low and it's almost time for my chron.

I have noticed that sometimes I've marked tasks complete and then they didn't save and I have to mark them again.

The weirdest thing about when my character died the last time was that it popped up a message that said -50 health. So that was the strangest part to me. That it showed for some reason that it all went away at the same time.
